src/app.ts

```diff
@@ -1,6 +1,72 @@
+type ResultData = {
+    id: string,
+    gender: string,
+    age: number,
+    row: number,
+}
+
+type PagingData = {
+    next?: string,
+    previous?: string,
+}
+
+let previousBtn: HTMLElement | any = document.querySelector('button[data-prevbtn]');
+let nextBtn: HTMLElement | any = document.querySelector('button[data-nextbtn]');
```

Comment:
Codacy has a fix for the issue: [Replace `'button[data-nextbtn]'` with `"button[data-nextbtn]"`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
let nextBtn: HTMLElement | any = document.querySelector("button[data-nextbtn]");
```